index.html

```diff
@@ -83,5 +83,270 @@ <h1 class="text-dark hero-title">
         </button>
       </div>
     </section>
+
+    <section class="work-section">
+      <header>
+        <h1 class="section-header text-center">My Recent Works</h1>
+        <span class="underline"></span>
+      </header>
+
+      <div class="work-lists">
+        <div class="box flex flex-col">
+          <div class="box-img">
+            <img
+              class="w-full h-full"
+              src="./assets/images/img-placeholder.png"
+              alt="imgPlaceholder"
+            />
+          </div>
+          <div class="box-content fw-600">
+            <h2 class="">Multi-Post Stories Gain + Glory</h2>
+            <div class="tags flex w-full justify-between">
+              <span>Ruby on Rails</span>
+              <span>css</span>
+              <span>JavaScript</span>
+              <span>HTML</span>
+            </div>
```

Comment:
- [ ] Please make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") is a list `ul` and each of them is a `li` element for semantic reasons. :pray: